server/routes/message.js

```diff
@@ -0,0 +1,9 @@
+const express = require("express");
+const router = express.Router({ mergeParams: true });
+const messageController = require("../controllers/messageController");
+
+// api/conversations/:convoID/messages
+router.post("/create", messageController.create);
+router.get("/", messageController.read);
```

Comment:
To best follow REST API naming conventions, this should be the same end point, just using a GET or a POST to differentiate what we should do

Reply:
Good point, have fixed routes to reflect this.